src/bootstrap/doc.rs

```diff
@@ -196,7 +196,7 @@ impl Step for RustbookSrc {
 
         let vers = match self.version {
             RustbookVersion::MdBook1 => "1",
-            RustbookVersion::MdBook2 => "2",
+            RustbookVersion::Latest => "3",
```

Comment:
Hm, I am not totally sure about the "latest" name, given that it's not literally the latest. It's probably fine.